Source/bmalloc/bmalloc/BumpAllocator.h

```diff
@@ -56,12 +56,7 @@ class BumpAllocator {
     unsigned m_remaining;
 };
 
-inline BumpAllocator::BumpAllocator()
-    : m_ptr()
-    , m_size()
-    , m_remaining()
-{
-}
+inline BumpAllocator::BumpAllocator() = default;
```

Comment:
This looks incorrect since `m_ptr`, `m_size,` `m_remaining` are left uninitialized.